exercises/H-array-literals/exercise.js

```diff
@@ -4,8 +4,8 @@
   Declare some variables assigned to arrays of values
 */
 
-var numbers = []; // add numbers from 1 to 10 into this array
-var mentors; // Create an array with the names of the mentors: Daniel, Irina and Rares
+var numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; // add numbers from 1 to 10 into this array
```

Comment:
Here you want to create a function that adds these numbers to any array. So something like 
```
function addNumbersToArray(array){

}
```
Using a `for` loop here could help